modin/core/io/text/text_file_dispatcher.py

```diff
@@ -994,22 +1002,39 @@ def _read(cls, filepath_or_buffer, **kwargs):
         )
         if not use_modin_impl:
             return cls.single_worker_read(
-                filepath_or_buffer, callback=cls.read_callback, **kwargs
+                filepath_or_buffer, callback=read_callback, **kwargs
+            )
+
+        is_quoting = kwargs["quoting"] != QUOTE_NONE  # QUOTE_MINIMAL
```

Comment:
why do you want to move code around?

Reply:
This is a refactoring - making the code easier to read (variables are declared closer to where they are used), not directly related to the current issue.

Reply:
I would rather refactoring be done in a refactoring PR, not in a feature PR. If possible please revert.

Reply:
ok!